scripts/docker-compose.oidc-security.yaml

```diff
@@ -0,0 +1,182 @@
+version: "3.9"
```

Comment:
I see a couple of issues here:

1. We want to reduce the number of compose files back to a single one as new users of the repo find it hard to find the right file.
2. There is no explanation and link that this is related to vc authn. There is documentation for plain Keycloak: https://github.com/hyperledger-labs/business-partner-agent/blob/master/docs/concepts/security_keycloak.md  but no example compose file, while here it is the other way round
3. Without the knowledge on how to setup vc-authn just having this single file is only meaningful to you but everyone else has no clue what is going on.


So if this is necessary we need to find a better structure for this for example

-scripts
-- scenarios
--- plain-keykloak
--- keycloak-vc-authn

Another option would be to assume that all of this config happens somewhere else maybe in the helm chart repo.

Reply:
added some readme and organised the code by use. .

Reply:
Like we discussed on the last weekly meeting we plan to get rid of the many compose files and switch to compose profiles. As that many files cause a lot of confusion and are hard to maintain. So be aware that the files in your PR will get deleted again. But for now it's ok to merge them as they live in a sub folder.